src/mobility/types.ts

```diff
@@ -0,0 +1,102 @@
+export interface Translation {
+    language: string
+    value: string
+}
+
+export interface TranslatedString {
+    translation: Translation[]
+}
```

Comment:
Er man garantert å få en string `Translation`? Da kunne man kanskje gjort noe sånt for å slippe å håndtere `undefined`.
```suggestion
export interface TranslatedString {
    default: Translation
    translation: Translation[]
}
```

Reply:
APIet garanterer at man får minst ein string, ja, på minst eitt språk. Så eg tenker me ikkje treng å innføre noko nytt. Det er fint at det er 1-1 med GraphQL-typen.